app/models/story.rb

```diff
@@ -2,33 +2,41 @@ class Story < ApplicationRecord
   validates :title, presence: true
 
   belongs_to :project
-
   has_many :estimates
+  has_many :users, through: :estimates
 
   before_create :add_position
 
-  scope :by_position, -> { order("position ASC NULLS FIRST, created_at ASC") }
+  scope :by_position, -> { order("stories.position ASC NULLS FIRST, stories.created_at ASC") }
```

Comment:
this is needed because using `includes(:estimates)` creates amiguity of which `created_at` column to sort by